github/activity_events.go

```diff
@@ -328,3 +330,30 @@ func (s *ActivityService) ListUserEventsForOrganization(ctx context.Context, org
 
 	return events, resp, nil
 }
+
+// RevokedAppAuthorizationEvent checks if authorization for an application has been revoked.
+//
+// Note if your GitHub App continues to use a revoked access token,
+// it will receive the 401 Bad Credentials error.
+//
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#githubappauthorizationevent
+func (s *AuthorizationsService) RevokedAppAuthorizationEvent(ctx context.Context, user string, opt *ListOptions) ([]*Event, *Response, error) {
```

Comment:
Please move this function and its corresponding test to the `authorizations.go` and `authorizations_test.go` files, respectively.

Reply:
Thank you @gmlewis for the review. I've addressed your comment with those changes.